pkgs/development/idris-modules/build-idris-package.nix

```diff
@@ -35,6 +36,7 @@
   '';
 
   installPhase = ''
```

Comment:
Likewise you won't need this.

Reply:
Do you mean the ```installPhase``` or just the ```export``` line?

Reply:
Just the export line. Guess I highlighted the wrong one sorry.